frontend/packages/console-app/src/components/nodes/NodeTerminal.tsx

```diff
@@ -144,26 +154,29 @@ const NodeTerminal: React.FC<NodeTerminalProps> = ({ obj: node }) => {
       deleteNamespace(namespace.metadata.name);
     };
     const createDebugPod = async () => {
+      // see https://github.com/openshift/oc/blob/fe18ae25a123e896fd218210073141f7f5811465/pkg/cli/debug/debug.go#L1267
       try {
         namespace = await k8sCreate(NamespaceModel, {
           metadata: {
             generateName: 'openshift-debug-',
             labels: {
-              'openshift.io/run-level': '0',
               'pod-security.kubernetes.io/audit': 'privileged',
               'pod-security.kubernetes.io/enforce': 'privileged',
               'pod-security.kubernetes.io/warn': 'privileged',
               'security.openshift.io/scc.podSecurityLabelSync': 'false',
             },
             annotations: {
               'openshift.io/node-selector': '',
+              'oc.openshift.io/command': 'oc debug',
             },
           },
         });
+        setActiveNamespace(namespace.metadata.name);
```

Comment:
Hmm, I'm trying to decide if this is good or not. Can I ask what the reason is for changing the active namespace? And do we delete the namespace after the pod terminates? Does that leave the user with an invalid active namespace?